k8s.gcr.io/ensure-prod-repo.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/sh
+#
+# This script creates & configures the "real" serving repo in GCR.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_PATH=$(readlink -f "$0")
+SCRIPT_DIR=$(dirname "$SCRIPT_PATH")
+source "${SCRIPT_DIR}/lib.sh"
+
+function usage() {
+    echo "usage: $0" > /dev/stderr
```

Comment:
Nit: we should use `>&2` here instead of `/dev/stderr` (https://unix.stackexchange.com/a/274044).

Reply:
ACK in next push